### PR TITLE
Refresh benchmark with columnar data

### PR DESCRIPTION
**Pull request by TonyXiang8787**
Do not pay attention to the result yet. It was a trial run. Please review on the code.

### CODE REVIEW

File: generate_fictional_dataset.py [hunk 1]
```diff
@@ -135,18 +135,8 @@ def generate_fictional_grid(
...
-    # add sym load
-    pgm_dataset["sym_load"] = pgm.initialize_array("input", "sym_load", n_load)
-    pgm_dataset["sym_load"]["id"] = np.arange(n_node + n_line, n_node + n_line + n_load, dtype=np.int32)
-    pgm_dataset["sym_load"]["node"] = pgm_dataset["node"]["id"][1:]
-    pgm_dataset["sym_load"]["status"] = 1
-    pgm_dataset["sym_load"]["type"] = pgm.LoadGenType.const_power
-    pgm_dataset["sym_load"]["p_specified"] = np.random.uniform(
-        low=load_p_w_min / 3.0, high=load_p_w_max / 3.0, size=n_load
-    )
-    pgm_dataset["sym_load"]["q_specified"] = pgm_dataset["sym_load"]["p_specified"] * np.sqrt(1 - pf**2) / pf
```
Comment (mgovers) [inline generate_fictional_dataset.py:148]: was removing this intended?

**Review by mgovers — APPROVED** [review]
code review LGTM. as you requested, I did not dive into numerical results. Feel free to resolve my comment and merge

File: generate_fictional_dataset.py (hunk 1 above)
Comment (TonyXiang8787) [reply generate_fictional_dataset.py:148]: Yes. This was a bug.

**Comment by TonyXiang8787** [comment]
@mgovers please review on the values

File: Power Grid Model Benchmark.ipynb [hunk 2]
```diff
@@ -1073,38 +9966,38 @@
...
+       "      <td>5.894751e-10</td>\n",
+       "      <td>3.375877e-09</td>\n",
        "      <td>4.059398e-06</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>N-1 symmetric calculation</th>\n",
-       "      <td>1.632916e-12</td>\n",
-       "      <td>5.740408e-13</td>\n",
+       "      <td>1.634692e-12</td>\n",
+       "      <td>6.065148e-13</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:9994]: not sure if it's significant, but it is a difference

File: Power Grid Model Benchmark.ipynb [hunk 3]
```diff
@@ -1113,24 +10006,24 @@
...
-       "Asymmetric calculation                          8.192199e-09   \n",
-       "Time series symmetric calculation               1.255385e-12   \n",
-       "Time series asymmetric calculation              3.375882e-09   \n",
-       "N-1 symmetric calculation                       5.740408e-13   \n",
-       "N-1 asymmetric calculation                      8.192231e-09   \n",
+       "Symmetric calculation                           5.166423e-13   \n",
+       "Asymmetric calculation                          8.192067e-09   \n",
+       "Time series symmetric calculation               1.300737e-12   \n",
+       "Time series asymmetric calculation              3.375877e-09   \n",
+       "N-1 symmetric calculation                       6.065148e-13   \n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10021]: 5% regression

File: Power Grid Model Benchmark.ipynb [hunk 4]
```diff
@@ -1193,74 +10086,74 @@
...
        "      <td>inf</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>Symmetric calculation without solver initialization</th>\n",
-       "      <td>0.000190</td>\n",
-       "      <td>0.000284</td>\n",
-       "      <td>0.000564</td>\n",
-       "      <td>0.000742</td>\n",
-       "      <td>0.013096</td>\n",
+       "      <td>0.000296</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10098]: not sure if significant, but it is a regression

File: Power Grid Model Benchmark.ipynb [hunk 5]
```diff
@@ -1193,74 +10086,74 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>Symmetric calculation with solver initialization</th>\n",
-       "      <td>0.000650</td>\n",
-       "      <td>0.000702</td>\n",
-       "      <td>0.000815</td>\n",
-       "      <td>0.000686</td>\n",
-       "      <td>0.018638</td>\n",
+       "      <td>0.000826</td>\n",
+       "      <td>0.001251</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10090]: not sure if significant, but it is a regression

File: Power Grid Model Benchmark.ipynb [hunk 6]
```diff
@@ -1193,74 +10086,74 @@
...
-       "      <td>0.001394</td>\n",
-       "      <td>0.002811</td>\n",
-       "      <td>0.001418</td>\n",
-       "      <td>0.594590</td>\n",
-       "      <td>0.018144</td>\n",
+       "      <td>0.001525</td>\n",
+       "      <td>0.001365</td>\n",
+       "      <td>0.003647</td>\n",
+       "      <td>0.001905</td>\n",
+       "      <td>0.914279</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10111]: interesting to see panda power almost 2x slower than last time

File: Power Grid Model Benchmark.ipynb [hunk 7]
```diff
@@ -1193,74 +10086,74 @@
...
-       "      <td>0.000795</td>\n",
-       "      <td>0.002003</td>\n",
-       "      <td>0.000924</td>\n",
-       "      <td>0.137374</td>\n",
-       "      <td>0.001243</td>\n",
+       "      <td>0.001026</td>\n",
+       "      <td>0.000964</td>\n",
+       "      <td>0.003057</td>\n",
+       "      <td>0.001343</td>\n",
+       "      <td>0.313200</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10120]: panda power regressed even more here

File: Power Grid Model Benchmark.ipynb [hunk 8]
```diff
@@ -1348,96 +10241,96 @@
...
-       "      <td id=\"T_2c730_row0_col1\" class=\"data row0 col1\" >26.55x</td>\n",
-       "      <td id=\"T_2c730_row0_col2\" class=\"data row0 col2\" >22.86x</td>\n",
-       "      <td id=\"T_2c730_row0_col3\" class=\"data row0 col3\" >27.17x</td>\n",
-       "      <td id=\"T_2c730_row0_col4\" class=\"data row0 col4\" >1.00x</td>\n",
-       "      <td id=\"T_2c730_row0_col5\" class=\"data row0 col5\" >0.00x</td>\n",
+       "      <th id=\"T_727bf_level0_row0\" class=\"row_heading level0 row0\" >Symmetric calculation with solver initialization</th>\n",
+       "      <td id=\"T_727bf_row0_col0\" class=\"data row0 col0\" >23.92x</td>\n",
+       "      <td id=\"T_727bf_row0_col1\" class=\"data row0 col1\" >15.80x</td>\n",
+       "      <td id=\"T_727bf_row0_col2\" class=\"data row0 col2\" >14.43x</td>\n",
+       "      <td id=\"T_727bf_row0_col3\" class=\"data row0 col3\" >20.63x</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10262]: some regression compared to the previous run, but not too much

File: Power Grid Model Benchmark.ipynb [hunk 9]
```diff
@@ -1348,96 +10241,96 @@
...
-       "      <td id=\"T_2c730_row1_col1\" class=\"data row1 col1\" >46.12x</td>\n",
-       "      <td id=\"T_2c730_row1_col2\" class=\"data row1 col2\" >23.22x</td>\n",
-       "      <td id=\"T_2c730_row1_col3\" class=\"data row1 col3\" >17.65x</td>\n",
-       "      <td id=\"T_2c730_row1_col4\" class=\"data row1 col4\" >1.00x</td>\n",
-       "      <td id=\"T_2c730_row1_col5\" class=\"data row1 col5\" >0.00x</td>\n",
+       "      <th id=\"T_727bf_level0_row1\" class=\"row_heading level0 row1\" >Symmetric calculation without solver initialization</th>\n",
+       "      <td id=\"T_727bf_row1_col0\" class=\"data row1 col0\" >45.41x</td>\n",
+       "      <td id=\"T_727bf_row1_col1\" class=\"data row1 col1\" >43.75x</td>\n",
+       "      <td id=\"T_727bf_row1_col2\" class=\"data row1 col2\" >24.25x</td>\n",
+       "      <td id=\"T_727bf_row1_col3\" class=\"data row1 col3\" >32.42x</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10271]: interesting how some have slowed down, but others have sped up relatively to PP

File: Power Grid Model Benchmark.ipynb [hunk 10]
```diff
@@ -1348,96 +10241,96 @@
...
-       "      <td id=\"T_2c730_row2_col3\" class=\"data row2 col3\" >419.35x</td>\n",
-       "      <td id=\"T_2c730_row2_col4\" class=\"data row2 col4\" >1.00x</td>\n",
-       "      <td id=\"T_2c730_row2_col5\" class=\"data row2 col5\" >32.77x</td>\n",
+       "      <th id=\"T_727bf_level0_row2\" class=\"row_heading level0 row2\" >Asymmetric calculation with solver initialization</th>\n",
+       "      <td id=\"T_727bf_row2_col0\" class=\"data row2 col0\" >599.37x</td>\n",
+       "      <td id=\"T_727bf_row2_col1\" class=\"data row2 col1\" >669.71x</td>\n",
+       "      <td id=\"T_727bf_row2_col2\" class=\"data row2 col2\" >250.72x</td>\n",
+       "      <td id=\"T_727bf_row2_col3\" class=\"data row2 col3\" >479.83x</td>\n",
+       "      <td id=\"T_727bf_row2_col4\" class=\"data row2 col4\" >1.00x</td>\n",
+       "      <td id=\"T_727bf_row2_col5\" class=\"data row2 col5\" >37.17x</td>\n",
```
Comment (mgovers) [inline Power Grid Model Benchmark.ipynb:10282]: seems like the slowing is caused by PP regressions, but we are benefiting much more than OpenDSS

**Review by mgovers — COMMENTED** [review]
reviewing shows a consistent pattern when diffing against the previous run:
* for symmetric calculations, the difference between PGM and PP is smaller than it used to be
* for asymmetric calculations, the difference seems to have increased (which is good for us)

**Comment by TonyXiang8787** [comment]
@mgovers now re-run without verbose. I think we can merge it. The numbers are within the run-variations for PGM. The reason why `pandapower` asymmetric is slower is unknown. They are slow anyway...

**Pull request merged**